tests/dumb_tests/test1.py

```diff
@@ -0,0 +1,9 @@
+from devito import Grid, TimeFunction,  Eq, Operator
```

Comment:
I'd rename this file as ``test_ops.py`` and I'd add a test ``test_init`` which simply captures an exception raised by ``ops.Operator._specialize_iet``, as discussed in our meeting yesterday